src/database/GroupsTable.cpp

```diff
@@ -60,14 +60,22 @@ bool Self::create()
     }
 }
 
-static GroupHandler readGroup(const QSqlQuery &query)
+GroupHandler Self::readGroup(const QSqlQuery &query, const QString &preffix)
 {
+    auto queryValue = [&query, &preffix](auto &name) {
+        if (preffix.isEmpty()) {
+            return query.value(name);
+        }
+        QString newName(name);
+        newName[0] = newName[0].toUpper();
+        return query.value(preffix + newName);
+    };
 
-    auto id = GroupId(query.value("id").toString());
-    auto superOwnerId = UserId(query.value("superOwnerId").toString());
-    auto name = query.value("name").toString();
-    auto cache = query.value("cache").toString();
-    auto invitationStatus = GroupInvitationStatusFromString(query.value("invitationStatus").toString());
+    auto id = GroupId(queryValue("id").toString());
+    auto superOwnerId = UserId(queryValue("superOwnerId").toString());
+    auto name = queryValue("name").toString();
```

Comment:
Move this lines after check ids to avoid crashes on invalid invitation status conversion.

Reply:
Done